src/blosc2/lazyexpr.py

```diff
@@ -103,6 +131,27 @@ def __getitem__(self, item):
         -------
         out: np.ndarray
             An array with the data containing the slice evaluated.
+
+        Examples
+        --------
+        >>> import blosc2
+        >>> import numpy as np
+        >>> dtype = np.float64
+        >>> shape = [30, 4]
+        >>> size = shape[0] * shape[1]
+        >>> a = np.linspace(0, 10, num=size, dtype=dtype).reshape(shape)
+        >>> b = np.linspace(0, 10, num=size, dtype=dtype).reshape(shape)
+        >>> #  Convert numpy arrays to Blosc2 arrays
+        >>> a1 = blosc2.asarray(a)
+        >>> b1 = blosc2.asarray(b)
+        >>> # Perform the mathematical operation
+        >>> expr = a1 + b1  # LazyExpr expression
+        >>> output = expr.eval(cparams={"typesize": 4})
+        >>> output[3]
```

Comment:
You are not checking getitem for `blosc2.NDArray` here, but rather for `np.array`.